tools/tune/eq/example_spk_eq.m

```diff
@@ -134,9 +127,9 @@ function example_spk_eq()
                 assign_response, ...
                 [ bq_fir ]);
         bp_fir = eq_fir_blob_pack(bm_fir);
-        eq_alsactl_write('example_spk_eq_fir.txt', bp_fir);
-        eq_blob_write('example_spk_eq_fir.blob', bp_fir);
-	eq_tplg_write('example_spk_eq_fir.m4', bp_fir, 'FIR');
+        eq_alsactl_write(fn_fir_sofctl, bp_fir);
+        eq_blob_write(fn_fir_blob, bp_fir);
+	eq_tplg_write(fn_fir_tplg, bp_fir, fir_priv, fir_comment);
```

Comment:
@singalsu minor nti: indentation seems off here

Reply:
Yep, there's mix of tabs and spaces and Github shows non even line starts while emacs doesn't. There would be more white space changes to fix this.